### ISSUE TITLE
Krill update failed

### ISSUE THREAD

**Issue by opensourcegal** [NLnetLabs/krill#671]
Updated Krill from 0.8 via apt (ubuntu focal).

Krill refuses to start afterwards: 

Krill failed to start: Could not upgrade Krill: Unknown key: ca-51/info.json

I digged a bit around and moved the info.json from data directory ca-51/migration/info.json to ca-51/info.json but then it refused to start on ca-50/info.json and on thenext try again on ca-51/info.json

**Comment by timbru** [comment]
That is unfortunate. We tested these upgrades multiple time, so I am not sure what went wrong. It looks like Krill was stopped before completing the data migration, and then on a subsequent start expected to find certain files to archive for 
the migration (to the migration subdir) which had already been moved.

It's hard to debug without more context.

Can you share the logs with me from the time of the first upgrade? Perhaps it's easier to handle this via email. You can email me at: tim _at_ nlnetlabs.nl - or if you are on the [rpki discord channel](https://discord.com/invite/8dvKB5Ykhy) we can have a 1-1 chat there.

**Comment by opensourcegal** [comment]
You were absolutely right and your comment made me finish the upgrade without problems.

Actually the log entry was:

[INFO] Will migrate 44675 commands for CA 

expect to finish: about 20 minutes later

So that was a really long migration process and on my first try I restarted the system for a kernel update which interrupted the migration process. After krill did not came back, I rolled back to a snapshot of the system from before the upgrade.

So a good advise for people running "bigger" CAs: Do a snapshot before the upgrade in case something like a power outage happens.

Maybe this can be avoided by making the upgrade resumable or all/nothing in the future. But that is really a low priority.

@timbru Many thanks for your fast answer, your work can only be described as outstanding!
